zebra/zebra_srv6_vty.c

```diff
@@ -938,6 +940,40 @@ DEFPY (locator_behavior,
 	return CMD_SUCCESS;
 }
 
+DEFPY (locator_flavor_psp,
+       locator_flavor_psp_cmd,
+       "[no] flavor psp",
+       NO_STR
+       "Configure SRv6 flavors\n"
+       "Specify Penultimate Segment Popping flavor\n")
+{
+	VTY_DECLVAR_CONTEXT(srv6_locator, locator);
+
+	if (no && !CHECK_FLAG(locator->flags, SRV6_LOCATOR_PSP))
+		/* SRv6 locator PSP flag already unset, nothing to do */
+		return CMD_SUCCESS;
+
+	if (!no && CHECK_FLAG(locator->flags, SRV6_LOCATOR_PSP))
+		/* SRv6 locator PSP flag already set, nothing to do */
+		return CMD_SUCCESS;
+
+	if (!locator->sid_format)
```

Comment:
Guard `if (!locator->sid_format)` should be removed.

We have a keword `format usid-f3216` that is used to specify the SID format. For example:
```
segment-routing
 srv6
  locators
   locator MAIN
    prefix fcbb:bb01:1::/48
    format usid-f3216
```

When keyword `format usid-f3216` is specified under the locator, `locator->sid_format` will be not NULL.

We still want to send locator del / locator add notifications to zclients, so that zclients can uninstall/withdraw the SID and install/advertise the SID with the correct codepoint (e.g. uN PSP).